components/eam/src/physics/cam/clubb_intr.F90

```diff
@@ -1795,7 +1799,20 @@ subroutine clubb_tend_cam( &
        rvm(i,k)     = state1%q(i,k,ixq)
        um(i,k)      = state1%u(i,k)
        vm(i,k)      = state1%v(i,k)
+
+#define NEWTHETAL
+#ifndef NEWTHETAL
        thlm(i,k)    = state1%t(i,k)*exner_clubb(i,k)-(latvap/cpair)*state1%q(i,k,ixcldliq)
+#else
+!NCAR
+!       thlm(i,k) = ( state1%t(i,k) &
+!                     - (latvap/cpairv(i,k,lchnk))*state1%q(i,k,ixcldliq) ) &
+!                   * inv_exner_clubb(i,k)
+
+       thlm(i,k) = ( state1%t(i,k) &
+                     - (latvap/cpair)*state1%q(i,k,ixcldliq) ) &
+                   * exner_clubb(i,k)
```

Comment:
Is this so-called "exner_clubb" actually the inverse of the Exner function?  If so, the variable name is confusing.

Reply:
yes, is has been like this in EAM:
```
       exner_clubb(i,k) = (real(p0_clubb, kind = r8 )/state1%pmid(i,k))**(rair/cpair)
```
I believe it is fixed in SCREAM.

Reply:
To clarify, changing how Exner is defined would be out of scope of this PR.